packages/suite/src/views/wallet/trading/DCA/TradingDCALanding.tsx

```diff
@@ -76,83 +61,211 @@ const StoreBadge = styled.div<{ $isLight: boolean }>`
                             <Translation id="TR_TRADING_DCA_HEADING" />
                         </H2>
                         <Column gap={spacings.xxl} alignItems="start">
-                            <FeatureItem icon="shieldCheck" featureNumber={1} />
-                            <FeatureItem icon="arrowDown" featureNumber={2} />
-                            <FeatureItem icon="lightning" featureNumber={3} />
-                            <FeatureItem icon="eye" featureNumber={4} />
+                            <FeatureItem icon="suitcaseSimple" featureNumber={1} />
+                            <FeatureItem icon="shieldCheck" featureNumber={2} />
+                            <FeatureItem icon="handCoins" featureNumber={3} />
                         </Column>
-                    </Column1>
-                    <Column2>
-                        <Image image="TRADING_INVITY_ICON" width={48} height={48} />
-                        <WrappedText>
-                            <Paragraph typographyStyle="highlight">
-                                <Translation id="TR_TRADING_DCA_DOWNLOAD" />
-                            </Paragraph>
-                        </WrappedText>
-                        <TrezorLink href={TRADING_DOWNLOAD_INVITY_APP_URL}>
-                            <Image image="TRADING_DCA_INVITY_APP_QR" width={170} height={170} />
-                        </TrezorLink>
-                        <Row>
-                            <TrezorLink href={TRADING_DOWNLOAD_INVITY_APP_URL}>
-                                <StoreBadge $isLight={isLightTheme}>
-                                    <Image image="PLAY_STORE_TITLE" height={26} />
-                                </StoreBadge>
-                            </TrezorLink>
-                            <StoreSeparatorWrapper>
-                                <Divider
-                                    orientation="vertical"
-                                    strokeWidth={1}
-                                    margin={{ left: spacings.sm, right: spacings.sm }}
-                                />
-                            </StoreSeparatorWrapper>
-                            <TrezorLink href={TRADING_DOWNLOAD_INVITY_APP_URL}>
-                                <StoreBadge $isLight={isLightTheme}>
-                                    <Image image="APP_STORE_TITLE" height={26} />
-                                </StoreBadge>
-                            </TrezorLink>
-                        </Row>
-                    </Column2>
+                    </Column>
+                    <Column padding={spacings.xxl} justifyContent="center" maxWidth={500}>
+                        <Image
+                            image={
+                                isLightTheme
+                                    ? 'TRADING_DCA_INVITY_APP'
+                                    : 'TRADING_DCA_INVITY_APP_DARK'
+                            }
+                            isFilterActive={false}
```

Comment:
Aha, so this needs to be reverted , should be left to default (changing to false from true)